matrix_client/api.py

```diff
@@ -326,6 +338,12 @@ def get_emote_body(self, text):
             "body": text
         }
 
+    def get_notice_body(self, text):
```

Comment:
I feel like this might be a tad overkill. I prefer functions to either do some processing, or have multiple uses ideally. Could this not be put inside send_notice?